src/Cake.Common.Tests/Fixtures/Tools/DotNetCore/DotNetCoreFixture.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Cake.Core.IO;
+using Cake.Core.Tooling;
+using Cake.Testing.Fixtures;
+
+namespace Cake.Common.Tests.Fixtures.Tools.DotNetCore
+{
+    internal abstract class DotNetCoreFixture<TSettings> : DotNetFixture<TSettings, ToolFixtureResult>
```

Comment:
This class could inherit directly from `ToolFixture<TSettings, ToolFixtureResult>`.